### ISSUE TITLE
Add support for YAML as output

### ISSUE THREAD

**Issue by OmarElabd** [OmarElabd/ObjectExporter#38]
requested by a user on extension site

**Comment by kwhitefoot** [comment]
I agree with the requester.  YAML would be very good because it is especially well suited to diff-ing, especially when compared to XML, because it has very little noise.
